data/battle_scripts_1.s

```diff
@@ -20,6 +20,36 @@
 
 	.section script_data, "aw", %progbits
 
+BattleScript_EffectSpicyExtract::
+	attackcanceler
+	jumpifsubstituteblocks BattleScript_ButItFailed
+	accuracycheck BattleScript_ButItFailed, NO_ACC_CALC_CHECK_LOCK_ON
```

Comment:
Are you sure this is `BattleScript_ButItFailed` and not the miss script? Because it can still miss because of semi-invulnerability. Also, I wouldn't hardcode the battle script itself to never miss, that should be something set in the move data. Because say there's another spicy extract clone in the future that does have an actual accuracy check.

Reply:
Yes, I agree. I'm sorry, this should have been obvious.